Makefile

```diff
@@ -8,7 +8,7 @@ sdist:
 	find `pwd`/dist -name '*.tar.gz' -printf '%f\n' -exec mv {} . \;
 
 tarball:
-	git archive --prefix=osbuild-$(VERSION)/ --format=tar.gz HEAD > $(VERSION).tar.gz
+	git archive --prefix=osbuild-$(VERSION)/ --format=tar.gz HEAD > $(PACKAGE_NAME)-$(VERSION).tar.gz
```

Comment:
Same question, this is just for consistency with the name we get when downloading from github through rpm, or is it important for the rest of the commit?

Reply:
no, just for `make rpm` to work